pyproject.toml

```diff
@@ -181,7 +182,6 @@ tdengine = [
     "taos-ws-py>=0.3.8"
 ]
 teradata = ["teradatasql>=16.20.0.23"]
-thumbnails = ["Pillow>=10.0.1, <11"]
```

Comment:
@michael-s-molina is the removal of an optional dependency considered a breaking change? As `Pillow` is hereby promoted to a hard dependency, do we need to leave the `thumbnails` optional dependency empty until the breaking window?

Reply:
I think it is a breaking change but we're kind of are in a breaking window based on @rusackas' [email](https://lists.apache.org/thread/pmb6z6lspt4322nonm8vrc1h8gxfdpb1)?. I'm not the release manager for 6.0 so @sadpandajoe @rusackas will probably know if we still are in a breaking window or not.

Reply:
@michael-s-molina @villebro According to @rusackas email we're not accepting new breaking changes now, so I set `thumbnails = []` in `pyproject.toml`.
Also added an entry in `UPDATING.md`